datadog/runchecks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo `date`;
+. ~/.bashrc;
+~/q/l32/q ./datadog/datachecks.q
```

Comment:
What is this file? The version and location of q should not be hardcoded as it will differ between hosts.